### ISSUE TITLE
Try again to bring Harbor-CLI into Homebrew

### ISSUE THREAD

**Issue by Vad1mo** [goharbor/harbor-cli#99]
> this is a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

In https://github.com/Homebrew/homebrew-core/pull/165860 the PR to add harbor CLI to Homebrew was rejected, we should try this again, once we reach a certain maturity and recognition in the community.

**Comment by tylerauerbeck** [comment]
@Vad1mo if nothing else, would it be worth setting up a tap for this in the `goharbor` org so folks can pull it from there via `brew`?

**Comment by Vad1mo** [comment]
yes this is the goal, to have a tap set up. We need to make the first relese of that

**Comment by Vad1mo** [comment]
@tylerauerbeck I have no preference, for now it has to be a tap. Whatever move us closer to a release pipeline is good enough

**Comment by bupd** [comment]
closing in favor of #296
